apps/src/StudioApp.js

```diff
@@ -15,8 +15,8 @@ var FeedbackUtils = require('./feedback');
 * The minimum width of a playable whole blockly game.
 */
 var MIN_WIDTH = 900;
-var MIN_MOBILE_SHARE_WIDTH = 450;
-var MOBILE_NO_PADDING_SHARE_WIDTH = 400;
+var MOBILE_SHARE_WIDTH_PADDING = 50;
```

Comment:
It's hard to infer the effect of this, but does this mean we want to allow mobile displays as narrow as 50px?